data/ecosystems/p/polkadot.toml

```diff
@@ -5330,6 +5332,9 @@ url = "https://github.com/Polkadex-Platform/Polkadex"
 [[repo]]
 url = "https://github.com/Polkadex-Substrate/Polkadex"
 
+[[repo]]
+url = "https://github.com/Polkadot-Heroes/Members"
+
```

Comment:
you can remove these lines since the taxonomy operates like a graph and will upstream the subecosystem's repos/devs